js/script.js

```diff
@@ -393,14 +388,11 @@ var RightClick = RightClick || {};
         return true;
     };
 
-    exports.isAMenuOpened = function (key) {
-        for (key; key < exports.menus.length; key++) {
+    exports.isAMenuOpened = function () {
+        for (let key = 0; key < exports.menus.length; key++) {
```

Comment:
is this code compiled afterwise? Because this is not compatible with ie11 :(

Reply:
Ah yes, it is not, but it should ^^

Thx ! :)